internal/commands/ip_access/create_inputs.go

```diff
@@ -0,0 +1,22 @@
+package ipaccess
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+type createInputs struct {
+	cli.ProjectInputs
+	IPAddress  string
+	Comment    string
+	UseCurrent bool
+	AllowAll   bool
+}
+
+func (i *createInputs) Resolve(profile *user.Profile, ui terminal.UI) error {
+	if err := i.ProjectInputs.Resolve(ui, profile.WorkingDirectory, false); err != nil {
+		return err
+	}
+	return nil
```

Comment:
do you need to do anything if the `IPAddress` or `Comment` field is still left blank, prompt for any values and/or error here?